Sources/LowEnergyCommandParameter.swift

```diff
@@ -1240,11 +1240,10 @@ public extension LowEnergyCommand {
         
         public var byteValue: [UInt8] {
             
-            let peerIdentifyAddressTypeBytes = peerIdentifyAddressType.rawValue.littleEndian
```

Comment:
@jmarkstar You don't need to convert a byte to little endian since a single byte doesn't contains endianness. Endianness only applies to multi-octet value types (e.g. `UInt16`).

Reply:
ok